cns/service/main.go

```diff
@@ -87,6 +89,11 @@ const (
 
 	// envVarEnableCNIConflistGeneration enables cni conflist generation if set (value doesn't matter)
 	envVarEnableCNIConflistGeneration = "CNS_ENABLE_CNI_CONFLIST_GENERATION"
+	// envVarEnableAsyncPodDelete enables CNS and CNI to use fsnotify package for async pod deletes
+	envVarEnableAsyncPodDelete = "ASYNC_POD_DELETE"
+
+	cnsBaseURL    = "http://localhost:10090"
```

Comment:
this is the [default](https://github.com/Azure/azure-container-networking/blob/e102891d98372e28a5e39ba235e02039677e877a/cns/client/client.go#L63), so we don't need it. if we instantiate the client with `""` it will use this

Reply:
alternatively we could make the default `client.defaultURL` public -> `client.DefaultURL` and pass it in where we construct the client. then we're not polluting `main` with these vars/consts that don't really belong in the root context